CommerceCloud/Integration/03_SalesforceParticularities.md

```diff
@@ -0,0 +1,170 @@
+
+When a new site is subsequently created, the corresponding site-specific attribute is automatically created after the connection has been synchronized.
+
+All attributes must be created in Salesforce and cannot be created in Actindo and dynamically be added to Salesforce. The attribute definitions list displays all available attributes in Salesforce:    
+*Administration > System Object Types > Product > Tab Attribute Definitions*   
+The attributes that are created in the *Omni-Channel* module as available attributes are imported from this attribute definitions list.
+
+
+## Default language
+
+In Salesforce, a default language is preconfigured and other languages can be added. All languages beside the default language are mapped via their key. Therefore, the language key in Actindo must equal the language key in Salesforce.   
+As there exists no default language in Actindo, a language must be specified as default language when configuring the connection settings, see [Configure the Salesforce connection](./02_ManageSalesforceConnection.md#configure-the-salesforce-connection). The language defined in the settings will be mapped with the default language in Salesforce as well as with the corresponding language in Salesforce.
+
+
+## Site-specific status
+
+An offer in Actindo can have one of the following three statuses:      
+- **Active**   
+    The offer is active. It is displayed in the marketplace, where it can be sold.   
+- **Inactive**   
+    The offer is inactive.  It is not longer displayed in the marketplace and thus cannot be sold anymore.   
+- **Not available**   
+    The offer is deleted from the marketplace.
+
+By default, the status of an offer applies to the offer in all sites. As there exists the *Override online (only active products) status for Site "site ID"* site-specific attribute, it is possible to override the *active* status of an offer for the specific site. That means, an active offer can be switched to inactive for the individual sites.   
+Note that the site-specific status change is only possible in one direction: If the status of an offer is set to **inactive**, it will be inactive on all sites and cannot be changed to **active** for single sites. 
+
+
+## Price books
+
+In Salesforce, all prices are saved in so-called *Price Books*. IN contrast to Actindo, the price is not an attribute in Salesforce, but a completely own entity which is largely detached from the product.   
+As price books in Salesforce cannot be accessed via API, the IDs of the price books as well as the ISO-code of the corresponding currency must be specified when configuring the connection settings, see [Configure the Salesforce connection](./02_ManageSalesforceConnection.md#configure-the-salesforce-connection). The price book IDs are displayed in the *ID* column of the price books list in Salesforce:   
```

Comment:
The attributes for newly added price books are added after syncing the connection

Reply:
meinst du damit die price book ids als destination attribute im ETL-Mapping?